core/plugins/org.polarsys.capella.core.data.core.validation/plugin.xml

```diff
@@ -152,8 +152,11 @@
                      class="Constraint">
                </target>
                <description>
-                  This rule ensures that a constraint constraining elements is not stored unexpectedly. 
-                  (A constraint shall not be stored under a part if not constraining parts or under a deployment link if not constraining deployment links.)
+                  In monopart mode: This rule ensures that a constraint is not stored under and have ConstrainedElements values as a Part/PartDeploymentLink.
+                  Exception : In EPBS Layer constraint's can be stored under Part
+
+                  In multipart mode : This rule ensures that a constraint is not stored under PartDeploymentlink.
+                  Exception : If first ConstraintElements value is container PartDeploymentlink or empty ConstraintElements value
```

Comment:
If a constraint is not constraining any element or if its stored under and constraining only a PartDeploymentLink